frontend/src/components/UserCard.js

```diff
@@ -30,7 +30,7 @@ export default class UserCard extends Component {
 
     const href = `https://opencollective.com/${user.username}`;
 
-    const roleLabel = formatAnchor(user.tier || user.role);
+    const roleLabel = formatAnchor(user.role || user.tier);
```

Comment:
uggh we are introducing another string in the mix that's going to need to be tracked on frontend and backend separately? Why do we even need `core contributor`? Isn't it same as `member` everywhere?